src/app/story/directives/status.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { cit, create, provide, stubPipe } from '../../../testing';
 import { Router } from '@angular/router';
 import { Angulartics2 } from 'angulartics2';
 import { RouterStub } from '../../../testing/stub.router';
-import { ModalService, ToastrService } from '../../core';
+import { ToastrService, ModalService } from '../../core';
```

Comment:
I've got ToastrService coming up behind this one, so I'll address it here with ModalService and make sure Toastr follows the same, but I'll leave component module export/import in shared and core until we prove otherwise.